astronomy-pipeline/astronomy_load.py

```diff
@@ -0,0 +1,55 @@
+"""Load functions to upload nested list data to relevant tables."""
+
+from psycopg2.extras import execute_values
+
+from astronomy_extract_functions import get_db_connection
+
+
+def upload_body_position_data(body_data: list[list]) -> None:
+
+    # Positions list order [timestamp, distance_km, azimuth,
```

Comment:
Several examples for this pipeline, but the comments that give examples of example inputs is very useful, makes the function more readable and saves time when reviewing. Much appreciated.